Misc/NEWS.d/next/Library/2023-04-14-21-16-05.gh-issue-103548.lagdpp.rst

```diff
@@ -0,0 +1,3 @@
+Improve performance of :meth:`pathlib.PurePath.is_absolute` and
```

Comment:
Maybe mention that the performance improvement for `is_absolute` only applies for non-Windows builds?